api/novelist.py

```diff
@@ -461,54 +464,83 @@ def get_isbns_from_query(self, library):
         i2 = aliased(Identifier)
 
         isbnQuery = select(
-            [i1.identifier, i1.type, i2.identifier]
+            [i1.identifier, i1.type, i2.identifier,
+            Edition.title, Edition.medium, Edition.sort_author, Edition.author,
```

Comment:
I got both `sort_author` and `author` because in the test `Edition.sort_author` was empty. So just covering the case where it could be empty from the table. Does that sound right?

Reply:
If you're getting the contributions (which is better anyway) then you don't need Edition.sort_author and Edition.author at all.